.github/workflows/ci.yml

```diff
@@ -8,25 +8,29 @@ on:
 
 jobs:
   cabal:
-    name: ${{ matrix.os }} / ghc ${{ matrix.ghc }}
+    name: cabal / ghc-${{ matrix.ghc }} / ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest]
-        cabal: ["latest"]
+        os:
+          - ubuntu-latest
+          # - macOS-latest
```

Comment:
I removed osx since I don't have any way of testing it or fixing problems locally.  Also, for packages like `pretty-simple` with few system-level non-haskell dependencies, it is rare to find a problem that only happens on a single OS.